substrate/frame/benchmarking/pov/src/lib.rs

```diff
@@ -40,6 +40,12 @@ pub mod pallet {
 	pub trait Config: frame_system::Config {
 		#[allow(deprecated)]
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+
+		/// The expected storage root hash for the benchmarking pallet.
+		///
+		/// Can be set to `None` to disable the check.
+		#[pallet::constant]
+		type StorageRoot: Get<Option<Vec<u8>>>;
```

Comment:
Can we please remove this "weird change" and then in a follow up provide a better test that reproduces the actual problem.